src/View/Helper/TimeHelper.php

```diff
@@ -203,7 +228,7 @@ public function toAtom($dateString, $timezone = null)
      */
     public function toRss($dateString, $timezone = null)
     {
-        $timezone = $timezone ?: date_default_timezone_get();
+        $timezone = $timezone ?: $this->defaultOutputTimezone;
```

Comment:
RSS strings should always end up in UTC. I don't think using the defaultOutputTimezone is correct here.